rfcs/0000-backup-and-restore.md

```diff
@@ -0,0 +1,44 @@
+---
+feature: backup-and-restore
+authors:
+  - "zwang28"
+start_date: "2022/11/14"
+---
+
+# Backup and Restore
+
+## Motivation
+Support backup and restore to protect against data loss, due to user error, software error, disaster scenario, etc.
+
+## Design
+
+A RisingWave cluster's persistent data includes metadata stored in meta store, and SSTable stored in object store. We back up these types of data differently:
+- For metadata, we back up a metadata snapshot. See [Metadata Snapshot](#metadata-snapshot).
+- For SSTable, we decide **not to replicate object store data** during backup. Instead, we only back up a SSTable manifest, which references to the same SSTable as the running cluster. See [SSTable Maintenance](#sstable-maintenance).
```

Comment:
What if users hope to backup data to other storage?

Reply:
I think we can optionally support incremental SSTable replication.